tests/test_utils/gcp_system_helpers.py

```diff
@@ -32,43 +31,6 @@
 )
 
 
-SKIP_TEST_WARNING = """
-The test is only run when the test is run in environment with GCP-system-tests enabled
-environment. You can enable it in one of two ways:
-
-* Set GCP_CONFIG_DIR environment variable to point to the GCP configuration
-  directory which keeps the {} key.
-* Run this test within automated environment variable workspace where
-  config directory is checked out next to the airflow one.
-
-"""
-
-SKIP_LONG_TEST_WARNING = """
-The test is only run when the test is run in with GCP-system-tests enabled
-environment. And environment variable GCP_ENABLE_LONG_TESTS is set to True.
-You can enable it in one of two ways:
-
-* Set GCP_CONFIG_DIR environment variable to point to the GCP configuration
-  directory which keeps variables.env file with environment variables to set
-  and keys directory which keeps service account keys in .json format and
-  set GCP_ENABLE_LONG_TESTS to True
-* Run this test within automated environment variable workspace where
-  config directory is checked out next to the airflow one.
-"""
-
-
-LOCAL_EXECUTOR_WARNING = """
```

Comment:
Don't we need it anymore? Do we have another way to run tests with LocalExecutor?

Reply:
Yes. I replaced it with @pytest.markers.backend("postgres", "mysql"). LocalExecutor is the default executor for both Postgres and Mysql  and SequentialExecutor is default (and the only available) for sqlite. By constraining the tests to run only on Postgres/MySql we automatically set local executor for unit tests.